LICENSE.TXT

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2009-2015, GATK Authors. All rights reserved.
+Copyright (c) 2009-2017, GATK Authors. All rights reserved.
```

Comment:
Same question here as in https://github.com/broadinstitute/gatk-protected/pull/1099 -- should the copyright holder be "Broad Institute, Inc.", or is "GATK Authors" correct?